### ISSUE TITLE
Add an ID or class to symbols-container

### ISSUE THREAD

**Issue by CollinY2K** [ivanvotti/ember-svg-jar#51]
When using the symbol strategy, it would be very useful to have an ID or class added on the SVG symbols-container.

**Why?**
As described in this [article](https://tympanus.net/codrops/2015/07/16/styling-svg-use-content-css/) in section _Overriding Presentation Attribute Values From CSS,_ you may wish to set the containing SVG path styles to `inherit` to enable cascading styles for the symbols contained within. Without an ID or class on the symbols-container, we're unable to target the collection of symbols and would instead have to specifically reference each individual symbol based on their ID. While workable, this is not ideal when you have a large collection.

**Comment by ivanvotti** [comment]
Do you mean something like this?

```javascript
{
  svgJar: {
    strategy: 'symbol',

    symbol: {
      containerAttrs: {
        id: 'my-container-id',
        width: '0',
        height: '0',
        xmlns: 'http://www.w3.org/2000/svg',
        'xmlns:xlink': 'http://www.w3.org/1999/xlink'
      },
      prefix: 'icon-',
      includeLoader: true
    }
  }
}
```

Then the result could look like:

```handlebars
  <svg id="my-container-id" width="0" height="0" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <symbol id="icon-foo" viewBox="0 0 16 16"><path d=""/></symbol>
    <symbol id="icon-bar" viewBox="0 0 24 24"><path d=""/></symbol>
  </svg>
```

**Comment by CollinY2K** [comment]
@ivanvotti Yes, precisely. I think your example is more comprehensive than my original suggestion of just providing an option to the svg container, but it would certainly address the issue.

**Comment by ivanvotti** [comment]
@CollinY2K The feature is in the master branch and will be available in the coming `1.0.1` release today. Here is how to use it: https://github.com/ivanvotti/ember-svg-jar/blob/master/docs/configuration.md#containerattrs
